### ISSUE TITLE
Changebar Issue

### ISSUE THREAD

**Issue by lagbox** [Komodo/KomodoEdit#1557]
### Short Summary

Depending upon where on the changebar clicked at times the popover log will not have a transparent background below where clicked on the sidebar.

### Steps to Reproduce

On a colored section in the changebar click somewhere below the top part of it.

### Expected results

Solid background

### Actual results

Transparent below a certain point.

### Platform Information

*Komodo Edit or IDE?* IDE
*Komodo Version?* 10.0.1, build 89160
*Operating System (and version)?* Ubuntu 15.10

### Additional Information

Where the background becomes transparent correlates to where I clicked the changebar.

![diff-log-interface-issue](https://cloud.githubusercontent.com/assets/3752215/15392907/1605351c-1d98-11e6-900e-c9c4fc7b8e78.png)

**Comment by Naatan** [comment]
Unable to reproduce. @mitchell-as could you try on Ubuntu? Sorry my VM is a PITA to use and it should only take a minute for you :)

**Comment by mitchell-as** [comment]
Yes I can reproduce this. I have a file with more than 100 consecutive lines marked "green" (added, I guess) and clicking on the changebar produces a half solid, half transparent window like in the original screenshot.

**Comment by Naatan** [comment]
Alright, thanks @mitchell-as :)
